src/openforms/prefill/sources.py

```diff
@@ -56,15 +56,18 @@ def invoke_plugin(
         item: tuple[BasePlugin, IdentifierRoles, list[dict[str, str]]],
     ) -> tuple[list[dict[str, str]], dict[str, JSONEncodable]]:
         plugin, identifier_role, fields = item
+        log = logger.bind(plugin=plugin, for_role=identifier_role, fields=fields)
 
         if not plugin.is_enabled:
+            log.debug("plugin_disabled")
             raise PluginNotEnabled()
 
         attributes = [attribute for field in fields for attribute in field]
+        log.debug("prefill.plugin.lookup_attributes", attributes=attributes)
         try:
             values = plugin.get_prefill_values(submission, attributes, identifier_role)
         except Exception as e:
-            logger.exception(f"exception in prefill plugin '{plugin_id}'")
+            log.exception("prefill.plugin.retrieve_failure")
```

Comment:
A bit confused again, I see we use `log.exception` and sometimes `log.error` when we have an Exception. What is the reason for this? In `logger.error` we also pass the exc so that's why I dont understand this.

Reply:
`log.exception` automatically inspects the frames and grabs the exception. it's not much more complicated than "I simply converted existing code" and the `.exception` communicates a higher severity than `.error` IMO so that's why I kept it.